libs/stats/odc/stats/_cli_run.py

```diff
@@ -36,7 +36,7 @@
     help="Mark outputs for public access (default: no)",
 )
 @click.option(
-    "--apply_eodatasets3/--not_apply_eodatasets3",
+    "--apply_eodatasets3",
```

Comment:
```suggestion
    "--apply-eodatasets3",
```
Click will still use the same variable name, but it's prettier and more consistent with other application options.